compiling/gdx-jnigen-gradle/src/main/java/com/badlogic/gdx/jnigen/gradle/JnigenExtension.java

```diff
@@ -258,6 +263,10 @@ private void checkForTasksToAdd (BuildTarget target) {
             jnigenBuildTask.setBuildTarget(target);
             jnigenBuildTask.dependsOn(jnigenTask);
 
+            JnigenSignTask jnigenSignTask = project.getTasks().create("jnigenSign" + os.name() + "_" + architecture.getDisplayName() + bitness.name(), JnigenSignTask.class, this);
+            jnigenSignTask.setBuildTarget(target);
+            jnigenSignTask.dependsOn(jnigenBuildTask);
```

Comment:
Could we maybe add the jnigenSign task conditionally, based on whether signing was configured? 
And if signing is configured, adding something like:
`jnigenBuildTask.finalizedBy(jnigenSignTask)`?
Conceptionally I feel like signing belongs more to the packaging step, but I guess that doesn't work well with the current jnigen setup that packages everything on a linux host.

Reply:
I added a check to add the tasks only when configured.
I don't really have an opinion on `finalizedBy` or `dependsOn`